.github/ISSUE_TEMPLATE/webtestplan.md

```diff
@@ -992,11 +988,44 @@ Add the following to enable read access to trusted clusters
     - [ ] macOS
     - [ ] Windows
     - [ ] Linux
-- [ ] Verify that logs are collected for all processes (main, renderer, shared, tshd) under
-      `~/Library/Application\ Support/Teleport\ Connect/logs`.
-- [ ] Verify that the password from the login form is not saved in the renderer log.
-- [ ] Log in to a cluster, then log out and log in again as a different user. Verify that the app
-      works properly after that.
-- [ ] Clean the Application Support dir for Connect. Start the latest stable version of the app.
-      Open every possible document. Close the app. Start the current alpha. Reopen the tabs. Verify that
-      the app was able to reopen the tabs without any errors.
+- VNet
+  - VNet doesn't work with local clusters made available under custom domains through entries in
+    `/etc/hosts`. It's best to use a "real" cluster. nip.io might work, but it hasn't been confirmed
+    yet.
+  - Verify that VNet works for TCP apps within:
+    - [ ] a root cluster
+    - [ ] a custom DNS zone of a root cluster
+    - [ ] a leaf cluster
+    - [ ] a custom DNS zone of a leaf cluster
+  - [ ] Verify that setting a custom IPv4 CIDR range works.
```

Comment:
Maybe we should link to vnet docs for custom DNS zones? It can be difficult to understand what exactly needs to be configured.

Reply:
I assumed that if someone doesn't know how to set up something, the first thing they'd do is look it up in the docs. ;f

The PR with docs hasn't been merged yet, but I linked to the target locations that will be available after the PR gets merged.